docker/shared/docker_config.ini

```diff
@@ -0,0 +1,23 @@
+[DOCKER]
+use_docker_config = False
+
+[DATABASE]
+db_host = db
+db_port = 5432
+db_user = fourcat
+db_name = fourcat
+db_password = supers3cr3t
```

Comment:
Would it be worth randomizing this password too, or is that unnecessary?

Reply:
The only issue with randomizing, is that it needs to be set before Docker creates the images. I am not sure the best way to do that, but can look into it. We could have the user set it themselves or notify them to go and update it.

I am also actually saving the admin password as a text file inside the docker image. I did that because, depending on a user's logging level, Docker does not always show the echo commands in the docker-entrypoint.sh file. I am not sure of a better solution at the moment.

Reply:
In that case a notification in the output for users to update it if they are concerned about security is probably enough - the database should not be available from outside anyway. A similar notification could then also be shown about login.txt, recommending users to delete it after they have properly saved the login somewhere else.

Reply:
Alright, for Docker you set the database info in one place: .env. That gets used when docker sets up the database, when the Dockerfile updates our config files and the username and database name are used by the entrypoint any time the container is run.

I updated the readme.md file to mention both deleting the file with the admin password (there is also a readout mentioning that) and tell the user where to go to set their database info.